### ISSUE TITLE
Can't get integer numbers on the Y axis

### ISSUE THREAD

**Issue by duzenko** [danielgindi/Charts#1150]
I need to display integer numbers on the Y axis but they shows as float (with trailing zeroes)
Code:
```
- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view, typically from a nib.
    
    LineChartView *chart = [[LineChartView alloc] init];
    int N = 5, M = 11;
    NSMutableArray *vals = [NSMutableArray arrayWithCapacity:N],
        *xVals = [NSMutableArray arrayWithCapacity:M];
    for (int i=0; i<M; i++) {
        if (i < N) {
            ChartDataEntry *entry = [[ChartDataEntry alloc] initWithValue:98+i xIndex:vals.count];
            [vals addObject:entry];
        }
        [xVals addObject:[NSNumber numberWithInt:i]];
    }
    LineChartDataSet *dataSet = [[LineChartDataSet alloc] initWithYVals:vals];
    LineChartData *data = [[LineChartData alloc] initWithXVals:xVals dataSet:dataSet];
    [chart setData:data];
    chart.leftAxis.valueFormatter.maximumFractionDigits = 0;
    chart.frame = CGRectMake(0, 30, self.view.frame.size.width, 100);
    [self.view addSubview:chart];

}
```
Result:

![simulator screen shot jun 17 2016 6 58 29 pm](https://cloud.githubusercontent.com/assets/1481807/16156651/e50ca052-34bd-11e6-8092-b22556096c4d.png)

**Comment by liuxuan30** [comment]
hi @duzenko why create a new issue as a duplicated one? I closed #1123 because I think it could be. If you got updates or questions for the same issue, you can directly post in the original one, when someone or I see it, we will update as well, so you don't have to create new ones.

As we discussed in #1123, you simply set maximumFractionDigits = 0 and it will work, I did tried it with ChartsDemo. As for you, could try ChartsDemo out? maximumFractionDigits is played in bar chart view controller, you can start from there, line 68.

**Comment by liuxuan30** [comment]
and be aware that, in your code, you never create a value formatter for left axis. setting any properties will take no effect at all.
Please check out ChartsDemo code and play with it to get warmed up.
